fge_bin_id: {bin1_1: 10, bin1_2: 9, bin1_3: 6, bin1_4: 7, bin1_5: 8, bin2_1: 3, bin2_2: 5,
  bin2_3: 4, bin2_4: 2, bin2_5: 1}
